ansible/roles/test/tasks/port_toggle.yml

```diff
@@ -0,0 +1,27 @@
+- name: turn off all ports on device
```

Comment:
are you only consider all ports on/off or more like link_flap validate per port so you won't turn off wrong port?

Reply:
As we discussed, we rely on the link_flap test to check individual link. This test is to stress the DUT.